app/src/components/modal/modal_deposit_withdraw/index.tsx

```diff
@@ -467,49 +377,38 @@ export const ModalDepositWithdraw = (props: Props) => {
                 />
 
                 <span>
-                  {currencySelected === ExchangeCurrency.Dai
-                    ? '0.00 DAI'
-                    : exchangeType === ExchangeType.withdraw
-                    ? `${formatNumber(formatBigNumber(displayFundAmount.div(1000), omenToken.decimals, 3), 2)} OMN`
-                    : '0.00 OMN'}
+                  {exchangeType === ExchangeType.withdraw && currencySelected !== 'dai'
+                    ? `${formatNumber(formatBigNumber(displayFundAmount.div(1000), decimals, 22), 3)} ${symbol}`
```

Comment:
Is this supposed to have 22 decimals precision?

Reply:
So I put so many decimals there because if its some small number and I put 2 decimals formatBigNumber will round it to 0 and then formatNumber won't recognize that its below 0.01 and it will just say 0.00 instead of <0.01....

Reply:
should I have done it differently?

Reply:
hey @Mi-Lan maybe try something like this?

```<span>
                  {currencySelected !== 'dai' && exchangeType === ExchangeType.withdraw
                    ? `${formatNumber(formatBigNumber(displayFundAmount, decimals, decimals))} ${symbol}`
                    : `${formatNumber(formatBigNumber(displayFundAmount, decimals, decimals))} ${symbol}`}
                </span>```
                
hope this is helpful!
                
![formatNumber](https://user-images.githubusercontent.com/64822997/120815422-caad2380-c51d-11eb-8faa-6f29b5acb7ca.gif)

Reply:
![formatNumber](https://user-images.githubusercontent.com/64822997/120815525-e6b0c500-c51d-11eb-9d12-fe5bfed14f5a.gif)